src/main/java/gov/nasa/pds/tools/label/LabelValidator.java

```diff
@@ -363,11 +366,20 @@ private Boolean determineSchematronValidationFlag(URL url) {
           //         bundle*.xml
           //         collection*.[xml,csv]
 
-          if (TargetExaminer.isTargetBundleType(url) || TargetExaminer.isTargetCollectionType(url)) {
+          // Remove time consuming check of reading the file in favor of checking the filename.
+          // The file name should be enough to know if it is a bundle or a collection.
+          validateAgainstSchematronFlag = false;
+          Matcher matcher = null;
+          matcher = BUNDLE_LABEL_PATTERN.matcher(FilenameUtils.getName(url.toString()));
```

Comment:
@qchaupds I don't think the string matching is a very expensive operation + it seems like we will continue to have nesting of checks here, but not a huge deal. we are a go for merge (after #396)